yarn.lock

```diff
@@ -0,0 +1,1906 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
scripty uses npm and an npm package-lock.json. Could you please use npm & update its package-lock? I'm uncomfortable with the idea of having two (potentially divergent) sources of truth as to the library's nested dependencies.

Reply:
Oops 😄 (total just assumed yarn... I wonder if there is a wrapper that would just delegate to either yarn or npm based on the presence of a yarn.lock or package-lock 🤔). I will get rid of this and use `package-lock.json`.